ibm/provider/provider.go

```diff
@@ -894,108 +894,107 @@ func Provider() *schema.Provider {
 			"ibm_is_bare_metal_server_network_interface":             vpc.ResourceIBMIsBareMetalServerNetworkInterface(),
 			"ibm_is_bare_metal_server":                               vpc.ResourceIBMIsBareMetalServer(),
 
```

Comment:
Please make sure you have removed only security_group_network_interface_attachment here..